File: 1. Getting Started/java/GcdAndLcm.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        // write your code here
        Scanner scn = new Scanner(System.in);
        int n1 = scn.nextInt();
        int n2 = scn.nextInt();
        int t1 = n1;
        int t2 = n2;
        while (t1 % t2 != 0) {
            int rem = t1 % t2;
            t1 = t2;
            t2 = rem;

        }
        int gcd = t2;
        int lcm = (n1 * n2) / gcd;
        System.out.println(gcd + "\n" + lcm);
    }
}

File: Getting Started/cpp/GcdAndLcm.cpp
#include <iostream>
using namespace std;
int gcd(int a,int b){
    if(b==0){
        return a;
    }
    return gcd(b,a%b);
}
int main(int argc, char **argv){
    int num1, num2;
    cin >> num1 >> num2;
    
    int y= gcd(num1,num2);
    int x=(num1*num2)/y;
    //write your code here
    cout << y << " "<< x <<endl;
    return 0;
}